backend/src/api.ts

```diff
@@ -250,6 +256,11 @@ loginCheckedPost('/hideShoutout', async (req, user) => {
   return {};
 });
 
+loginCheckedPost('/deleteShoutout', async (req, user) => {
```

Comment:
Out of curiosity, why is this a post request and not a delete request?

Reply:
I can change it, but several other deletes are also post requests, so I was trying to keep it standardized. For example, `deleteTeamEvent`, `deleteSignInForm`, and `deleteTeam`.

Reply:
Let's leave it as it is right now, we can change all the delete endpoints to DELETE requests if we decide to later on.

Reply:
OK, I’m good with leaving it.